evap/student/templates/student_index_semester_evaluations_list.html

```diff
@@ -0,0 +1,164 @@
+                                                            data-toggle="tooltip" data-placement="top" title="{{ grade_document.description }}">
+                                                            <span class="far fa-file"></span>
+                                                        </a>
+                                                    {% endfor %}
+                                                    {% if course.grade_documents.count > 1 %}
+                                                        </div>
+                                                    {% endif %}
+                                                {% endif %}
+                                            {% endif %}
+                                        </div>
+                                        {% if course.evaluation_count == 1 %}
+                                            <span class="badge badge-secondary">{{ course.type }}</span>
+                                        {% endif %}
+                                        {% if evaluation.is_midterm_evaluation %}<span class="badge badge-dark">{% trans 'Midterm evaluation' %}</span>{% endif %}
+                                        {% if evaluation.is_single_result %}
+                                            <span class="badge badge-success">{% trans 'Single result' %}</span>
+                                        {% endif %}
+                                        {% if course.evaluation_count == 1 %}
+                                            <i class="small">{{ course.responsibles_names }}</i>
+                                        {% endif %}
+                                    </td>
+                                    <td style="width: 20%">
+                                        {% if semester.results_are_archived %}
+                                        {% elif evaluation.is_single_result %}
+                                            <span data-toggle="tooltip" data-placement="top" title="{% trans 'Individual participants are not stored for single results. The number shown here is the number of votes received.' %}">
+                                                {% include 'progress_bar_fill.html' with icon='user' total=evaluation.single_result_rating_result.count_sum fill=100 %}
+                                            </span>
+                                        {% else %}
+                                            {% if evaluation.state == 'evaluated' or evaluation.state == 'reviewed' or evaluation.state == 'published' %}
+                                                {% include 'progress_bar.html' with icon='user' done=evaluation.num_voters total=evaluation.num_participants %}
+                                            {% elif evaluation.participates_in and not evaluation.voted_for %}
+                                            {% else %}
```

Comment:
I don't understand.

Reply:
similar to the case above. i will rewrite the conditions.